code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -35,41 +35,139 @@
 	id = "nicotine"
 	description = "Slightly reduces stun times. If overdosed it will deal toxin and oxygen damage."
 	reagent_state = LIQUID
-	color = "#60A584" // rgb: 96, 165, 132
-	addiction_threshold = 30
+	color = rgb(0,0,0)
+	overdose_threshold = 30
 
 /datum/reagent/drug/nicotine/on_mob_life(mob/living/M)
 	if(prob(1))
 		var/smoke_message = pick("You feel relaxed.", "You feel calmed.","You feel alert.","You feel rugged.")
 		M << "<span class='notice'>[smoke_message]</span>"
-	M.AdjustStunned(-1, 0)
+	if(prob(50))
+		M.Jitter(5)
```

Comment:
Changing the nicotine description text to reflect that it doesn't reduce stun times anymore will save some time in the future. Also, as a regular detective player this saddens me